config/routes.rb

```diff
@@ -71,9 +71,13 @@
 
   namespace :orga do
     resources :teams
-    resources :seasons
+    resources :seasons do
+      member do
+        get :switch_seasons
```

Comment:
From a RESTful perspective, it's more a PUT – but that's nit-picking so don't mind me :wink: